nemo/collections/nlp/metrics/sacrebleu.py

```diff
@@ -0,0 +1,1922 @@
+# Copyright 2020 NVIDIA. All Rights Reserved.
```

Comment:
do we actually need this file or we could rather take sacrebleu as a dependency?